src/components/Guilds/Web3Modals/WalletModal.tsx

```diff
@@ -43,7 +43,7 @@ const Divider = styled.hr`
 `;
 
 const GreyDivider = styled(Divider)`
-  border-top: 1px solid #c4c4c4;
+  border-top: 1px solid ${({ theme }) => theme.colors.muted};

```

Comment:
Not 100% from your pr, but we have a Divider component in components/Guilds/common that's doing the same thing. Maybe a good spot for components refactor.